mysql/errname.go

```diff
@@ -968,6 +968,21 @@ var MySQLErrName = map[uint16]string{
 	ErrColumnStateNonPublic:       "can not use non-public column",
 	ErrIndexStateCantNone:         "index %s can't be in none state",
 	ErrInvalidRecordKey:           "invalid record key",
+	ErrInvalidDDLWorker:           "Invalid DDL worker",
+	ErrUnsupportedDDLOperation:    "Unsupported",
```

Comment:
Maybe this name is too short? Unsupported what？

Reply:
Please check the comment above.